src/app/account/auth/login/page.tsx

```diff
@@ -50,16 +53,33 @@ export default function Login() {
           <Input
             label={"Password"}
             leftContent={<LockIcon size={24} />}
+            rightContent={
+              isPwdVisible ? (
+                <EyeSlashIcon
+                  size={24}
+                  onClick={() => setIsPwdVisible(!isPwdVisible)}
```

Comment:
Instead of duplicating the onClick logic, how about using a "button" (unstyled) to wrap it and apply the onClick once there? Then button will contain the conditional icon

Reply:
sure! updated at fd262bd87995bb4e44541bebfcc80421fda95412